starter-code/lib/viking.js

```diff
@@ -19,13 +19,78 @@
 //   FootViking: FootViking
```

Comment:
Good work.  It seems like you figured out how to pass all the test except the one you changed